x-pack/plugins/alerting/server/authorization/alerting_authorization.ts

```diff
@@ -382,47 +423,62 @@ export class AlertingAuthorization {
         kibana: [...privilegeToRuleType.keys()],
       });
 
+      let authorizedRuleTypes;
+      let unauthorizedRuleTypes;
```

Comment:
What is `unauthorizedRuleTypes` used for? It currently looks unused. If that is the case, can we remove?

Reply:
Removed for now. @dhurley14 may end up using it for the bulk update route, but we'll revisit once it's actually needed.